### PR TITLE
Rpc pr

### PR DESCRIPTION
**Pull request by ABD-AZE**
#61 execution.go exluded

### CODE REVIEW

**Comment by gerceboss** [comment]
@ABD-AZE , how clean does this look, good

**Comment by ABD-AZE** [comment]
> @ABD-AZE , how clean does this look, good

500 lines are from go.sum, just to clarify.

File: execution/rpc.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	SendRawTransaction(bytes *[]byte) (common.Hash, error)
+	GetTransactionReceipt(tx_hash *common.Hash) (types.Receipt, error)
+	GetTransaction(tx_hash *common.Hash) (types.Transaction, error)
+	GetLogs(filter *ethereum.FilterQuery) ([]types.Log, error)
+	GetFilterChanges(filer_id *uint256.Int) ([]types.Log, error)
+	UninstallFilter(filter_id *uint256.Int) (bool, error)
+	GetNewFilter(filter *ethereum.FilterQuery) (uint256.Int, error)
+	GetNewBlockFilter() (uint256.Int, error)
+	GetNewPendingTransactionFilter() (uint256.Int, error)
+    ChainId() (uint64, error)
```
Comment (gerceboss) [inline execution/rpc.go]: ensure liniting is maintained

File: execution/rpc.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	seleneCommon "github.com/BlocSoc-iitr/selene/common"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/holiman/uint256"
+)
+
+
+type ExecutionRpc interface{
+    New (rpc *string) (*ExecutionRpc, error)
```
Comment (gerceboss) [inline execution/rpc.go]: here too

**Pull request merged**